app/src/main/java/com/example/studentarena/EndlessRecyclerViewScrollListener.java

```diff
@@ -43,9 +43,12 @@ public int getLastVisibleItem(int[] lastVisibleItemPositions) {
             else if (lastVisibleItemPositions[i] > maxSize) {
                 maxSize = lastVisibleItemPositions[i];
             }
+            // if (i == 0 || lastVisibleItemPositions[i] > maxSize){
+            //  maxSize = Collections.max(lastVisibleItemPositions[i]);
```

Comment:
Having errors implementing this feedback

Reply:
no worries, we can chat about it tomorrow or at some later time. your code works, it's just doing something so common that i thought surely there was a ready-made solution you could use instead. there is, but using it is a little bit awkward, so no big deal if we make this something to revisit later